### PR TITLE
update: georgia code review

### PR DESCRIPTION
**Pull request by erika-tyagi**

### CODE REVIEW

File: georgia/covid-viz.R [hunk 1]
```diff
@@ -7,7 +7,7 @@ library(zoo)
 raw_dat <- behindbarstools::read_scrape_data(all_dates = TRUE, state = "Georgia")
 ga_state <- raw_dat %>%
     filter(Jurisdiction == "state",
-           Age != "Juvenile") %>%
+           !Age %in% c("Juvenile")) %>%
```
Comment (erika-tyagi) [inline georgia/covid-viz.R:10]: Original code dropped rows where `Age` is NA, new code keeps them. Went from 20,923 rows in `ga_state` to 21,467.

File: georgia/covid-viz.R [hunk 2]
```diff
@@ -28,9 +28,10 @@ latest_ga$Residents.Initiated / latest_ga$Residents.Population
 
 ## read statewide prison data and create metrics for analysis 
 historical_statewide <- read_csv("https://raw.githubusercontent.com/uclalawcovid19behindbars/data/master/historical-data/historical_state_counts.csv")
-ga_prison_pop <- behindbarstools::read_mpap_pop_data() %>%
-    filter(State == "Georgia") %>%
-    pull(Population.Feb20)
+
+# http://104.131.72.50:3838/scraper_data/raw_files/2020-03-01_historical_ga_pop.pdf
+ga_prison_pop <- 55221
+
```
Comment (erika-tyagi) [inline georgia/covid-viz.R:34]: Marshall Project data is from 4/1/2020 ([here](https://github.com/themarshallproject/COVID_prison_data/blob/master/data/prison_populations.csv)), not February 2020. Probably safer to use the number directly reported by the DOC, archived [here](http://104.131.72.50:3838/scraper_data/raw_files/2020-03-01_historical_ga_pop.pdf) for 3/1/2020 to have a safer source.

File: georgia/covid-viz.R (hunk 1 above)
Comment (hjohns12) [reply georgia/covid-viz.R:10]: Good catch, thank you!

File: georgia/covid-viz.R (hunk 2 above)
Comment (hjohns12) [reply georgia/covid-viz.R:34]: yesss thank you for this!

**Comment by hjohns12** [comment]
Thank you so much for going through this! Made updates in the script so closing this now.

**Pull request closed without merging**